gpp_parsed_consent.go

```diff
@@ -42,3 +78,113 @@ func ParseGppHeader(s string) (*GppHeader, error) {
 	g.Sections, _ = r.ReadFibonacciRange()
 	return g, r.Err
 }
+
+// ParseGpp takes a base64 Raw URL Encoded string which represents a GPP v1 string
+// of the format {gpp header}~{section 1}[.{sub-section}][~{section n}]
+// and returns each pair of section value and parsing function that should be used.
+// The pairs are returned to allow more control over how parsing functions are applied.
+func ParseGpp(s string) ([]GppSectionParser, error) {
```

Comment:
Could we rename this function to something more descriptive? idk something like `MapGppSectionToParser`, it would make it easier to differentiate between this and `ParseGppConsent`

Reply:
Yes, good call, updated!